### PR TITLE
fill missing crate info

### PR DESCRIPTION
**Pull request by cmeissl**
based on #5

### CODE REVIEW

File: libdisplay-info-derive/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,14 @@
 [package]
-name = "libdisplay-info-derive"
-version = "0.1.0"
+authors = ["Christian Meissl <meissl.christian@gmail.com>"]
+categories = ["api-bindings"]
+description = "EDID and DisplayID library."
```
Comment (YaLTeR) [inline libdisplay-info-derive/Cargo.toml:4]: I guess this should say this one's a utility crate

File: libdisplay-info-sys/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,15 @@
 [package]
-name = "libdisplay-info-sys"
-version = "0.1.0"
+authors = ["Christian Meissl <meissl.christian@gmail.com>"]
+categories = ["api-bindings"]
+description = "EDID and DisplayID library."
```
Comment (YaLTeR) [inline libdisplay-info-sys/Cargo.toml:4]: And this one should mention that it's the low level bindings

Comment (cmeissl) [reply libdisplay-info-sys/Cargo.toml]: thx, changed to low level bindings

File: libdisplay-info-derive/Cargo.toml (hunk 1 above)
Comment (cmeissl) [reply libdisplay-info-derive/Cargo.toml]: yeah, changed

**Pull request merged**